app/assets/javascripts/dashboard.js

```diff
@@ -64,34 +64,27 @@ function updateCurrentBookings(response) {
 }
 
 function getBackgroundColor(obj) {
-  var date = getDispalyedDate();
-  var newDate = new Date(date + " 00:00");
+  var currentDate = new Date(getDispalyedDate() + " 00:00");
   var color = (obj.state == 'booked') ? 'red' : 'green';
-  if (newDate < today()) {
+
+  if (currentDate < today()) {
     color = 'grey';
   }
+
   return color;
 }
 
-// function hideModal() {
-//   var date = getDispalyedDate();
-//   var newDate = new Date(date + " 00:00");
-//   if (newDate < today()) {
-//     $('#slot-modal').foundation('reveal', 'close');
-//   }
-// }
-
 function today() {
-  return new Date(new Date().toISOString().slice(0,10) + " 00:00")
+  return new Date(new Date().toJSON().slice(0,10) + " 00:00");
 }
 
 function getInfo(obj) {
-    var message = (obj.info.client != null) ? setSlotMessage(obj) : obj.info.time;
-    return message;
+  var message = (obj.info.client !== null) ? setSlotMessage(obj) : obj.info.time;
```

Comment:
@diraulo that `=`sign you added actually fks up everything and the resources don't display correctly anymore. It goes back to normal if I take it back. Is there a reason why we should keep it?